docs/meta/comparisons.md

```diff
@@ -50,13 +50,17 @@ than PyPDF2. See [history of pypdf](history.md).
   [QPDF]: https://github.com/qpdf/qpdf
 
 
-## pdfminer
+## pdfminer.six and pdfplumber
 
 [`pdfminer.six`](https://pypi.org/project/pdfminer.six/) is capable of
 extracting the [font size](https://stackoverflow.com/a/69962459/562769)
 / font weight (bold-ness). It has no capabilities for writing PDF files.
 
-## pdfrw / pdfminer / pdfplumber
+[`pdfplumber`](https://pypi.org/project/pdfplumber/) is a library focused on extracting data from PDF documents. Since `pdfplumber` is built on top of `pdfminer.six`, there are **no capabilities of exporting or modifying a PDF file** (see [#440 (discussions)](https://github.com/jsvine/pdfplumber/discussions/440#discussioncomment-803880)). However, `pdfplumber` is capable of converting a PDF file into an image, [draw lines and rectangles on the image](https://github.com/jsvine/pdfplumber#drawing-methods), and save it as an image file.
```

Comment:
> is capable of converting a PDF file into an image

From skimming the Readme, it looks like pdfplumber calls `Wand` for pdf rendering, which is a binding to `ImageMagick`, which in turn calls ghostscript for rendering, IIRC.
So this phrase is misleading as pdfplumber is not an actual pdf rendering library, but merely a rendering "wrapper-wrapper-wrapper".

Reply:
Yes, I agree! It is not a PDF rendering library, there's just one function to convert the PDF into an image with the tools you mentioned. I'm not experienced with `Wand`, `ImageMagick`, and `ghostscript`, so if you're an expert there, feel free to elaborate more on my changes.

Reply:
@RitchieP You could rephrase 

> However, `pdfplumber` is capable of converting a PDF file into an image

to

> However, `pdfplumber` is capable of converting a PDF file into an image via ImageMagick

Reply:
Definitely! I'll make a PR in a bit.

Reply:
Thanks!